scripts/genesis2db/main.go

```diff
@@ -88,7 +89,20 @@ func main() {
 		}
 	}
 
+	logrus.Info("import token")
 	i := 0
+	for k, v := range g.AppState.Token.Balances {
+		if err := s.AddPDV(context.Background(), k, v.Int64(), time.Time{}); err != nil {
```

Comment:
time.Now maybe? Another idea is to listen for the account first incoming send transaction and treat it as a date of registration.

Reply:
Not sure it's correct